packages/moonstone/Panels/ApplicationCloseButton.module.less

```diff
@@ -1,12 +1,5 @@
 // ApplicationCloseButton.module.less
 //
-@import '../styles/mixins.less';
-@import '../styles/variables.less';
 
 .applicationCloseButton {
```

Comment:
I can see some value in keeping this for future use but there would be a minor perf win for removing it by avoiding `styles` logic in `kind`.

Reply:
Yeah, this was one of the ways I thought of to solve the emptiness problem. I could also simply remove the import statement, comment out the import, or remove it _and_ delete the file. Do you have a preference?

Reply:
I don't think it'll move the needle much in either direction so I'll leave it to you to decide how to handle.